src/supervisor.tex

```diff
@@ -2261,3 +2281,108 @@ \chapter{``Svnapot'' Standard Extension for NAPOT Translation Contiguity, Versio
   configurations result in page-faults exceptions, while invalid access
   types or accesses to invalid physical memory regions trigger page faults.
 \end{commentary}
+
+\chapter{``Svpbmt'' Standard Extension for Page-Based Memory Attributes, Version 0.1}
+\label{svpbmt}
+
+{\bf Warning! This draft specification is likely to change before being
+accepted as standard by the RISC-V Foundation.}
+
+In Sv39, Sv48, and Sv57, bits 62--61 of the page table entry indicate the use
+of page-based memory types that override the PMA(s) for the associated memory
+pages.  The encoding for the PBMT bits is captured in Table~\ref{pbmt}.
+
+\begin{table*}[h!]
+\begin{center}
+\begin{tabular}{|r|l|}
+\hline
+Value  & Page-Based Memory Attributes \\
```

Comment:
To avoid any potential misunderstanding by a reader, it might be better to say "Page-Based Memory Attribute Combination".  (Which is what a "page-based memory type" is.)

Reply:
I'm not sure this is a "combination" either...it's really the "effective" memory attribute.  To your other comments, maybe we should use that term "effective memory attribute" throughout, so we can contrast it to "physical memory attribute" in a clearer way.